### DISCUSSION TITLE
Collection is empty in CP but exists in content->collections

### DISCUSSION THREAD

**Discussion by jayrdi** [statamic/cms#8508] [Q&A] [answered]
A collection of news articles I created a while back has suddenly disappeared, I'm not sure if this is as a result of clearing the stache..? I haven't edited these for a while, I just noticed when I went back to the homepage that the "news cards" were all missing.

I've attached a file that shows the empty collection in the CP and the actual .md files in the code...
![temp1](https://github.com/statamic/cms/assets/7912545/bc9c669c-8866-46d6-bfad-da2d0a08a9cc)
![temp2](https://github.com/statamic/cms/assets/7912545/1391a79b-d147-4721-b7dc-845c68268e8d)

**Comment by jasonvarga** [comment, marked as answer]
How weird! If you clear your cache it should come back. I'm not sure why it would ever disappear though. Any recent changes to the server? Are you using redis?

**Reply by jayrdi** [reply]
It's only local at the moment, running from vite

**Comment by duncanmcclean** [comment]
Can you share the contents of the entry files?

**Reply by jayrdi** [reply]
Clearing the cache fixed it although I've no idea what caused it to happen in the first place. Nothing weird in the entry files, just news articles with title, image, author, article text (
![Screenshot 2023-08-22 085719](https://github.com/statamic/cms/assets/7912545/52721a9d-051b-4d17-88ee-785efacbfef3)
e.g. attached)
